lib/wallaby/metadata.ex

```diff
@@ -1,9 +1,5 @@
 defmodule Wallaby.Metadata do
-  @moduledoc """
-  Metadata is used to encode information about the browser and test. This
-  information is then stored in a User Agent string. The information from the
-  test can then be extracted in the application.
-  """
```

Comment:
According to the link aaron posted to the elixir docs using code comments for private functions is the standard:

> ##  Documenting Private Functions
>
> Elixir warns if a private function has a @doc attribute and discards its content, because @doc is intended to be used only for your public interface.
>
> Private functions may still need internal documentation for maintainers, though. That can be accomplished with code comme